packages/sources/alpine/test/integration/README.md

```diff
@@ -0,0 +1 @@
+See `packages/source/curve/test/integration` and `packages/composite/synth-index/test/integration` for reference
```

Comment:
You'll want to flesh out some tests for this adapter. You can also look at the `chain-reserve-wallet` adapter for some examples of integration tests and how to get it working with mocks. Reason being is right now you're only testing 1 validation error, but not actually testing that the adapter works.

Reply:
If we have tests here we can delete this file